2-mandatory/1-recipes.js

```diff
@@ -21,5 +21,19 @@
 
   You should write and log at least 5 recipes
 */
+let TheRecipeCard = new Object;
+TheRecipeCard.title = "Items mix :D"
+TheRecipeCard.Serves = 2;
 
-// write code here
\ No newline at end of file
+
+TheRecipeCard.Ingredients = []
+
+let ingr = ["item1","item2","item3","item4"]
+
+ingr.forEach(elment => {
+  TheRecipeCard.Ingredients.push(elment)
+})
+console.log(TheRecipeCard.title)
+console.log(`Serves: ${TheRecipeCard.Serves}`)
+console.log(TheRecipeCard.Ingredients.forEach(property => {console.log(property)}))
+//why do I have underfind for 5th element wich doesn't exist ? :<
```

Comment:
The reason you're getting undefined is that the forEach method doesn't return a value. It simply iterates over each element of the array and performs an operation on it, but it doesn't return anything. for example: `TheRecipeCard.Ingredients.forEach(property => {
  console.log(property);`